### PR TITLE
Update Copyright end year

### PR DESCRIPTION
**Pull request by allevo**

### CODE REVIEW

File: src/website/layouts/partials/footer.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     {% if github_url %}
       <p><em>Found a typo or want to improve this page?</em> <a href="{{ github_url }}">Submit a PR on GitHub</a></p>
     {% endif %}
-      <p>Fastify, Copyright &copy; 2016-2017 Matteo Collina and the <a href="https://github.com/fastify/fastify#team" target="_blank">Fastify team</a>, Licensed under <a href="https://github.com/fastify/fastify/blob/master/LICENSE" target="_blank">MIT</a></p>
+      <p>Fastify, Copyright &copy; 2016-2018 Matteo Collina and the <a href="https://github.com/fastify/fastify#team" target="_blank">Fastify team</a>, Licensed under <a href="https://github.com/fastify/fastify/blob/master/LICENSE" target="_blank">MIT</a></p>
```
Comment (delvedor) [inline src/website/layouts/partials/footer.html]: As we did in the [LICENSE](https://github.com/fastify/fastify/blob/master/LICENSE) file we should write only "*The Fastify Team*" :)

Comment (allevo) [reply src/website/layouts/partials/footer.html]: Done

**Review by delvedor — APPROVED** [review]
LGTM

**Review by mcollina — APPROVED** [review]
LGTM

**Pull request merged**